webpack.config.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
         'exceljs': 'window.ExcelJS',
         'jspdf': 'window.jspdf.jsPDF',
         'devexpress-diagram': 'window.DevExpress.diagram',
-        'devexpress-gantt': 'window.DevExpress.Gantt'
+        'devexpress-gantt': 'window.DevExpress.Gantt',
+        'luxon': 'window.luxon'
```

Comment:
Why it is necessary?

Reply:
To avoid including luxon in our bundles (dx.all.js, dx.all.debug.js, etc).

Reply:
exclude `luxon` from bundle